polyfill/index.d.ts

```diff
@@ -215,7 +215,7 @@ export namespace Temporal {
     year(date: Temporal.Date): number;
     month(date: Temporal.Date): number;
     day(date: Temporal.Date): number;
-    era(date: Temporal.Date): unknown;
+    era(date: Temporal.Date): string | undefined;
```

Comment:
I had left `era` as `unknown` because of #526 — there is a problem with using strings for eras that I'm not sure is surmountable; and I hadn't originally intended to check in the Japanese calendar. That said, this change is fine with me because the TS types should reflect the current polyfill, but I just wanted to make sure you were aware of #526.

Reply:
Yep, I saw that & figured the design was a work in progress. I also noticed that `{era: undefined}`  in the current polyfill is returned in every `getFields()` call, even for non-ISO calendars' Date/DateTime/etc instances. Is this the intended long-term solution, or is the goal to only add custom calendar fields to instances that have those calendars?